src/components/Store.svelte

```diff
@@ -1,30 +1,42 @@
 <script>
+    import { createEventDispatcher } from 'svelte';
+    export let gold
+    const dispatch = createEventDispatcher();
+
     let products = [
-        { active: true, element: 'Apple', price: 1, level: 1 },
-        { active: true, element: 'Banana', price: 1, level: 1 },
-        { active: true, element: 'Wrench', price: 1, level: 1 },
-        { active: true, element: 'Hammer', price: 1, level: 1 },
-        { active: true, element: 'Knife', price: 1, level: 1 },
-        { active: true, element: 'Shirt', price: 1, level: 1 },
-        { active: true, element: 'Mallet', price:1, level: 1 },
+        {active: true, icon: '🍎', name: 'Apple', type: 'consumable', description: 'Recupera a vida.', attrib:{life:5}, price: 1, level: 1 },
+        {active: true, icon: '🍌', name: 'Banana', type: 'consumable', description: 'Recupera a vida.', attrib:{life:5},  price: 1, level: 1},
+        {active: true, icon: '🔧', name: 'Wrench', type: 'weapon', description: 'Equipamento.', attrib: {power:1},  price: 1, level: 1},
+        {active: true, icon: '🔨', name: 'Hammer', type: 'weapon', description: 'Equipamento.', attrib: {power:2},  price: 1, level: 1},
     ];
+
+    function handleBuyItem(product){
+        delete product.active
```

Comment:
Why are we removing this atribute before dispatching?

Reply:
Because we'll not be using it on inventory. It is used only to be displayed on store (like active=false is a secret item)

Reply:
If you see the itemsList (items the user already has in his inventory) you'll notice that these items objects don't have the active attribute because it is only used in Store Component. Tha said, to keep the inventory with the same objects(item) structures/pattern, I tought it was a good idea to delete this property before adding the new item to the inventory.